Common/UnitDefinitions/RotationalStiffnessPerLength.json

```diff
@@ -20,6 +20,30 @@
           "Abbreviations": [ "N·m/rad/m", "Nm/rad/m" ]
         }
       ]
+    },
+    {
+      "SingularName": "PoundForceFootPerDegreesPerFoot",
+      "PluralName": "PoundForceFeetPerDegreesPerFeet",
+      "FromUnitToBaseFunc": "x*254.864324570",
+      "FromBaseToUnitFunc": "x/254.864324570",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lbf·ft/deg/ft" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilopoundForceFootPerDegreesPerFoot",
+      "PluralName": "KilopoundForceFeetPerDegreesPerFeet",
+      "FromUnitToBaseFunc": "x*254864.324570",
+      "FromBaseToUnitFunc": "x/254864.324570",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kipf·ft/deg/ft" ]
```

Comment:
Same here, as per #749 :

1.) kipf·ft/deg/ft
2.) kip·ft/deg/ft
3.) k·ft/deg/ft
4.) kipf·ft/°/ft
5.) kip·ft/°/ft
6.) k·ft/°/ft

Although in this context, using the degree symbol in the middle looks weird to me?

Reply:
I guess, but I'd rather be consistent. See #749 about suggested ordering.
https://github.com/angularsen/UnitsNet/pull/749#discussion_r376120196